src/main.rs

```diff
@@ -86,7 +86,13 @@ fn main() -> anyhow::Result<()> {
             }
         }
         None => {
-            anyhow::bail!("subcommand required");
+            let check: cargo_semver_checks::Check = args.check_release.into();
+            let report = check.check_release()?;
+            if report.success() {
+                std::process::exit(0)
+            } else {
+                std::process::exit(1);
+            }
```

Comment:
Could we refactor the code here a bit to avoid the duplicated logic?

Maybe something like:
```rust
let check: cargo_semver_checks::Check = args.command.unwrap_or_else(|| args.check_release.into());

let report = check.check_release()?;
// ...
```